taca/utils/bioinfo_tab.py

```diff
@@ -116,17 +130,18 @@ def get_status(run_dir):
 
     return status
 
-
-
-
+#Returns project, FC, lane & sample (sample-run) status for a given folder
 def get_ss_projects(run_dir):
-    pattern=re.compile("(P[0-9]{3,5})_[0-9]{3,5}")
-    project_ids=set()
+    proj_tree = Tree()
+    proj_pattern=re.compile("(P[0-9]{3,5})_[0-9]{3,5}")
```

Comment:
what confounds me a bit is that we read a sample sheet and we know that it has a certain structure. But then instead of using the structure we guess field by field.
This was already implemented by @Galithil probably to not make any kind of guess over the SamplSheet structure. 
@Galithil what do you think?